README.md

```diff
@@ -11,13 +11,144 @@ Note: Currently kritis doesn't use grafeas and pulls vulnerability informtation
 
 Instructions for installing kritis on your cluster via helm can be found [here](https://github.com/grafeas/kritis/blob/master/kritis-charts/README.md).
 
+Installing Kritis, creates a number of resources in your cluster. Mentioned below are important ones.
```

Comment:
with the changes made for the single helm chart, it might make sense to mention all components somewhere else in the docs or here